metecho/api/jobs.py

```diff
@@ -709,7 +716,7 @@ def submit_review(*, user, task, data, originating_user_id):
         # GitHub has with emoji in status descriptions
         printable = set(string.printable)
         filtered_notes = "".join(filter(lambda c: c in printable, notes))
-        repository.create_status(
+        repo_as_app.create_status(
```

Comment:
Readonly users cannot update the PR status, so we use the App account to do that, but the user account to add any review comments:

![Screen Shot 2021-05-12 at 3 14 37 PM](https://user-images.githubusercontent.com/552316/118031739-df3e3780-b334-11eb-86a6-a896c5b5ddba.png)
![Screen Shot 2021-05-12 at 3 14 48 PM](https://user-images.githubusercontent.com/552316/118031741-dfd6ce00-b334-11eb-833f-c232ff414cca.png)